javascript-if/main.js

```diff
@@ -0,0 +1,75 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age < 16) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH >= 0 && pH < 7) {
+    return 'acid';
+  } else if (pH <= 14 && pH >= 8) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
+}
+
+function introduceWarnerBro(name) {
+  if (name === 'yakko' || name === 'wakko') {
+    return 'We\'re the warner brothers!';
+  } else if (name === 'dots') {
+    return 'I\'m cute~';
```

Comment:
It's `"dot"`, not `"dots"`:

---

![image](https://user-images.githubusercontent.com/16143771/169142041-8506f008-19bf-4818-a3a9-ac4f14aadea9.png)

Reply:
![image](https://user-images.githubusercontent.com/96744088/169142994-c893bb17-cd7e-40be-bfe4-89346dea17ec.png)
edited